chopper/test/test_service.dart

```diff
@@ -253,6 +253,15 @@ abstract class HttpTestService extends ChopperService {
     @Field() final List<int> positives, [
     @Field() final String? signature,
   ]);
+
+  @Get(path: 'get_timeout', timeout: Duration(seconds: 42))
+  Future<Response<String>> getTimeoutTest();
+
+  @Get(path: 'get_timeout_zero', timeout: Duration(seconds: 0))
```

Comment:
Not really familiar with timeouts but why would you need a timeout of `0`?

Reply:
Just a test :D